gen/dcos-config.yaml

```diff
@@ -1330,7 +1330,7 @@ package:
       GLOG_drop_log_memory=false
       MESOS_CGROUPS_ENABLE_CFS=true
       MESOS_CGROUPS_LIMIT_SWAP=false
-      MESOS_CONTAINERIZERS=docker,mesos
+      MESOS_CONTAINERIZERS=mesos,docker
```

Comment:
Since this is very subtle, I'd recommend a comment like
```
# See https://github.com/dcos/dcos/pull/4988 before changing this variable
```

Reply:
I was hoping the commit history would provide enough context in case anyone wants to change the line back.  (The PR description is taken from the commit description.)
But it certainly won't hurt to add.

Reply:
It would certainly provide enough context for anyone caring to look into the `git blame` for that line, but I can imagine e.g. someone re-ordering them alphabetically without looking at the history first because it's an 'obviously' harmless change.